REQUIRE

```diff
@@ -1,2 +1,3 @@
 julia 0.6
-Requires
+WinRPM
```

Comment:
```diff
diff --git a/REQUIRE b/REQUIRE
index d376198..98e2c02 100644
--- a/REQUIRE
+++ b/REQUIRE
@@ -1,3 +1,3 @@
 julia 0.6
-WinRPM
-Requires
\ No newline at end of file
+@windows WinRPM
+Requires
```

----

I got this

```julia
julia> Pkg.build("NNlib")           
ERROR: WinRPM is not an installed package                                
Stacktrace:                         
 [1] build!(::Array{String,1}, ::Set{Any}, ::String) at ./pkg/entry.jl:623                                                                         
 [2] build!(::Array{String,1}, ::Set{Any}, ::String) at ./pkg/entry.jl:624                                                                         
 [3] build!(::Array{String,1}, ::Dict{Any,Any}, ::Set{Any}) at ./pkg/entry.jl:635                                                                  
 [4] build(::Array{String,1}) at ./pkg/entry.jl:650                      
 [5] (::Base.Pkg.Dir.##4#7{Array{Any,1},Base.Pkg.Entry.#build,Tuple{Array{String,1}}})() at ./pkg/dir.jl:36                                        
 [6] cd(::Base.Pkg.Dir.##4#7{Array{Any,1},Base.Pkg.Entry.#build,Tuple{Array{String,1}}}, ::String) at ./file.jl:70                                 
 [7] #cd#1(::Array{Any,1}, ::Function, ::Function, ::Array{String,1}, ::Vararg{Array{String,1},N} where N) at ./pkg/dir.jl:36                      
 [8] build(::String, ::Vararg{String,N} where N) at ./pkg/pkg.jl:254
```

Reply:
You probably just need to do `Pkg.resolve()`. Although this should probably be `@windows WinRPM` @dfdx.

Reply:
Ah, right. Added it.